sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -803,6 +843,14 @@ class RocksDB(
   /** Get the write buffer manager and cache */
   def getWriteBufferManagerAndCache(): (WriteBufferManager, Cache) = (writeBufferManager, lruCache)
 
+  def getLatestCheckpointInfo(partitionId: Int): StateStoreCheckpointInfo = {
```

Comment:
Will this ever be called if `lastCommittedCheckpointId` is `None` or `LastCommitBasedCheckpointId` is `None`?